doc/devel/contributing.rst

```diff
@@ -227,8 +227,9 @@ rules before submitting a pull request:
   :file:`doc/users/next_whats_new/README.rst` for more information).
 
 * If you change the API in a backward-incompatible way, please document it in
-  :file:`doc/api/api_changes`, by adding to the relevant file
-  (see :file:`doc/api/api_changes.rst` for more information)
+  :file:`doc/api/next_api_changes/behaviour`, by adding a new file with the
+  naming convention ``20202-02-17-JMK.rst`` where the date is followed by the
```

Comment:
That's a bit too far in the future. Also maybe use the name of the template file.
```suggestion
  naming convention ``2020-01-01-AL.rst`` where the date is followed by the
```